host/config/config.json

```
@@ -1,12 +1,24 @@
+
 {
-    "bind": "0.0.0.0:8080",
-    "cache": "/tmp/sgx",
-    "log_path": "./log/raiko",
-    "concurrency_limit": 10,
-    "max_log_days": 7,
-    "log_level": "info",
     "network": "taiko_a7",
+    "rpc": "https://rpc.hekla.taiko.xyz/",
+    "l1_rpc": "https://l1rpc.hekla.taiko.xyz/",
+    "beacon_rpc": "https://l1beacon.hekla.taiko.xyz",
+    "block_no": 1,
+    "prover": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
+    "graffiti": "8008500000000000000000000000000000000000000000000000000000000000",
+    "proof_type": "sgx",
     "sgx": {
-        "instance_id": 0
+        "instance_id": 456,
+        "setup": true,
+        "bootstrap": true,
+        "prove": true,
```

```suggestion
        "setup": false,
        "bootstrap": false,
        "prove": true,
```
I think reasonable default values.